web/src/components/UserForm/UserForm.js

```diff
@@ -51,7 +62,8 @@ const UserForm = (props) => {
           className="rw-input"
           errorClassName="rw-input rw-input-error"
         />
-        <FieldError name="image" className="rw-field-error" />
+        <FieldError name="image" className="rw-field-error" /> */}
```

Comment:
Could you please remove the lines that you commented out?

Reply:
Okay sure